Update docs with getting help and add Cython dependencies

bocklund opened the pull request:
I'm not sure if the documentation commit should be here to automatically trigger the rebuild of the website. Should I just move the DOC commit over to my website branch and submit the PR to the pycalphad/website branch?

richardotis commented:
I think you did this correctly. The deployment script should automatically commit to the website branch after this commit is merged to develop.

richardotis commented on docs/index.rst:
```diff
@@ -37,6 +35,17 @@ A public chat channel is available on `Gitter`_.
+A public chat channel is available on `Gitter`_.
+The support email list can be reached at support@pycalphad.org.
+
+.. _Gitter: https://gitter.im/richardotis/pycalphad
```
This should be https://gitter.im/pycalphad/pycalphad

richardotis commented on docs/index.rst:
```diff
@@ -37,6 +35,17 @@ A public chat channel is available on `Gitter`_.
+Questions about installing and using pycalphad can be addressed in the `pycalphad Google Group`_.
+Technical issues and bugs should be reported on on `GitHub`_.
+A public chat channel is available on `Gitter`_.
+The support email list can be reached at support@pycalphad.org.
```
We cannot merge this until I've actually done something for email handling.

bocklund commented:
Fixed the link for gitter. I also changed the GitHub links to the pycalphad/pycalphad repo for clarity.
I left the email link in so we can merge whenever that is set up. I can remove it if you want to merge in the Cython dependencies now.

richardotis commented:
I'm +1 to removing the email link and merging now. The Google group will meet most users' needs, and those who want more privacy can find our respective email addresses without much effort.
